docs/en/observability/view-monitor-status.asciidoc

```diff
@@ -43,36 +43,47 @@ The information displayed includes the recent status of a host or endpoint, when
 was last checked, its URL, and, if applicable, the TLS certificate expiration time. There is
 also a sparkline showing downtime history.
 
+[TIP]
+=====
+Use monitor tags to display a custom assortment of monitors; for example, consider assigning
+tags based on a monitor's hosted cloud provider – making it easy to quickly see all monitors
+hosted on GCP, AWS, etc.
+=====
+
+Expand the table row for a specific monitor on the list to view additional
+information such as which alerts are configured for the monitor, a recent error and
+when it occurred, the date and time of any recent test runs, and it's URL.
+
 [role="screenshot"]
 image::images/monitors-list.png[Monitors list]
```

Comment:
Just to mention, the `monitors-list.png` screenshot will need to be recaptured when we have completed https://github.com/elastic/kibana/issues/88382 (as this was a confusing UI)

Reply:
I will keep an eye on that issue. Thanks.